Dockerfile

```diff
@@ -13,35 +13,58 @@ COPY \
 # /opt/mlserver/dist folder
 RUN ./hack/build-wheels.sh /opt/mlserver/dist
 
-FROM python:3.8-slim
+FROM registry.access.redhat.com/ubi9/ubi-minimal
 SHELL ["/bin/bash", "-c"]
 
+ARG PYTHON_VERSION=3.8.13
+ARG CONDA_VERSION=4.13.0
+ARG MINIFORGE_VERSION=${CONDA_VERSION}-1
 ARG RUNTIMES="all"
 
 ENV MLSERVER_MODELS_DIR=/mnt/models \
     MLSERVER_ENV_TARBALL=/mnt/models/environment.tar.gz \
-    PATH=/opt/mlserver/.local/bin:$PATH
+    MLSERVER_PATH=/opt/mlserver \
+    CONDA_PATH=/opt/conda \
+    PATH=/opt/mlserver/.local/bin:/opt/conda/bin:$PATH
```

Comment:
```suggestion
    PATH=${MLSERVER_PATH}/.local/bin:${CONDA_PATH}/bin:$PATH
```

Reply:
I tried this at first, but it seemed the Dockerfile didn't recognise them? Easiest was to just hardcode it and figure it out later.